pkg/installer/custominstallconfig_test.go

```diff
@@ -62,6 +66,9 @@ var expectedBootstrapStorageFileList = []string{"/etc/fluentbit/journal.conf",
 
 var expectedBootstrapSystemdFileList = []string{"fluentbit.service", "mdsd.service", "aro-etchosts-resolver.service", "dnsmasq.service"}
 
+var apiIntIP = "203.0.113.1"
+var expectedMasterIgnitionSource = "https://" + apiIntIP + ":22623/config/master"
```

Comment:
I am wondering where the port :22623 comes from and if it's also configurable?

Reply:
That is the port the Machine Config Server runs on, so it is [fixed](https://github.com/openshift/enhancements/blob/master/dev-guide/host-port-registry.md#tcp).